msbuild-api-docs/xml/Microsoft.Build.Experimental/OutOfProcServerNode.xml

```diff
@@ -0,0 +1,72 @@
+<Type Name="OutOfProcServerNode" FullName="Microsoft.Build.Experimental.OutOfProcServerNode">
```

Comment:
These experimental types are new. Is that expected?

Reply:
We should add the Experimental namespace to the filter

Reply:
The namespaces have been added to the _filter.xml.